vpd-manager/src/parser.cpp

```diff
@@ -346,4 +346,52 @@ int Parser::updateVpdKeywordOnHardware(
     return l_bytesUpdatedOnHardware;
 }
 
+int Parser::vpdSanityCheck() noexcept
+{
+    std::vector<types::Path> l_fruPaths{m_vpdFilePath};
```

Comment:
Do we need another data structure for this? Can we check it in place? I see Unnecessary string copies.